rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -328,9 +322,11 @@ def _convert_attribute_tokens_to_texts(
         attribute_texts = {}
 
         for attribute in attribute_tokens.keys():
-            attribute_texts[attribute] = [
-                " ".join(tokens) for tokens in attribute_tokens[attribute]
-            ]
+            tokens = attribute_tokens[attribute]
```

Comment:
let's call it `all_tokens` or smth, because it is a list of a list of tokens
```suggestion
            all_tokens = attribute_tokens[attribute]
```